qt/applications/workbench/workbench/plotting/toolbar.py

```diff
@@ -30,27 +30,27 @@ class WorkbenchNavigationToolbar(NavigationToolbar2QT):
     sig_waterfall_conversion = QtCore.Signal(bool)
 
     toolitems = (
-        ('Home', 'Center display on contents', 'mdi.home', 'on_home_clicked', None),
+        ('Home', 'Reset axes limits', 'mdi.home', 'on_home_clicked', None),
         ('Back', 'Back to previous view', 'mdi.arrow-left', 'back', None),
         ('Forward', 'Forward to next view', 'mdi.arrow-right', 'forward', None),
         (None, None, None, None, None),
-        ('Pan', 'Pan axes with left mouse, zoom with right', 'mdi.arrow-all', 'pan', False),
-        ('Zoom', 'Zoom to rectangle', 'mdi.magnify', 'zoom', False),
+        ('Pan', 'Pan: L-clk \nStretch: R-clk', 'mdi.arrow-all', 'pan', False),
+        ('Zoom', 'Zoom \n In: L-clk+drag \n Out: R-clk+drag', 'mdi.magnify', 'zoom', False),
         (None, None, None, None, None),
-        ('Grid', 'Toggle grid on/off', 'mdi.grid', 'toggle_grid', False),
-        ('Save', 'Save the figure', 'mdi.content-save', 'save_figure', None),
-        ('Print', 'Print the figure', 'mdi.printer', 'print_figure', None),
+        ('Grid', 'Grids on/off', 'mdi.grid', 'toggle_grid', False),
+        ('Save', 'Save image file', 'mdi.content-save', 'save_figure', None),
+        ('Print', 'Print image', 'mdi.printer', 'print_figure', None),
         (None, None, None, None, None),
-        ('Customize', 'Configure plot options', 'mdi.settings', 'launch_plot_options', None),
+        ('Customize', 'Options menu', 'mdi.settings', 'launch_plot_options', None),
         (None, None, None, None, None),
-        ('Create Script', 'Generate a script that will recreate the current figure',
+        ('Create Script', 'Generate Script to recreate the current figure',
```

Comment:
I think having capital letters anywhere but at the beginning looks strange.
```suggestion
        ('Create Script', 'Generate script to recreate the current figure',
```

Reply:
Okay, I was only using to highlight what I saw as the main word, but this is not a big thing so I can change this!